src/m/m_heap.cpp

```diff
@@ -215,7 +220,8 @@ EGG::ExpHeap *createDylinkHeap(size_t size, EGG::Heap *parent) {
     return g_dylinkHeap;
 }
 EGG::AssertHeap *createAssertHeap(EGG::Heap *parent) {
-    const char *name = "アサートヒープ(mHeap::assertHeap)";
+    // Help how do I tell dtk about string length?
```

Comment:
If this is extra space at the end of a section no need to worry, the next TU will align after iirc